### DISCUSSION TITLE
Welcome to CIPP Discussions!

### DISCUSSION THREAD

**Discussion by KelvinTegelaar** [KelvinTegelaar/CIPP#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DarrenChap** [comment]
Great work guys.  MSP in Australia here and this will make life so much easier!

**Comment by slsconsult** [comment]
What a piece of software 🗡️ 
Crazy work guys!
Greetings from Belgium

**Comment by dwoodmsp** [comment]
Is CIPP capable of reporting on available storage on a sharepoint site?

**Comment by initcyber** [comment]
Is CIPP capable of connecting to GCC/GCCH ?
